docs/clusterdefinition.md

```diff
@@ -1,559 +1,558 @@
-|"--cloud-provider"|"azure"|
-|"--cloud-config"|"/etc/kubernetes/azure.json"|
-|"--leader-elect"|"true"|
-|"--v"|"2"|
-
-<a name="feat-apiserver-config"></a>
-#### apiServerConfig
-
-`apiServerConfig` declares runtime configuration for the kube-apiserver daemon running on all master nodes. Like `kubeletConfig` and `controllerManagerConfig` it is a generic key/value object, and a child property of `kubernetesConfig`. An example custom apiserver config:
-
-```
-"kubernetesConfig": {
-    "apiServerConfig": {
-        "--request-timeout": "30s"
-    }
-}
-```
-Or perhaps you want to customize/override the set of admission-control flags passed to the API Server by default, you can omit the options you don't want and specify only the ones you need as follows:
-
-```
-"orchestratorProfile": {
-      "orchestratorType": "Kubernetes",
-      "orchestratorRelease": "1.8",
-      "kubernetesConfig": {
-        "apiServerConfig": {
-          "--admission-control":  "NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultStorageClass,ResourceQuota,AlwaysPullImages"
-        }
-      }
-    } 
-```
-
-See [here](https://kubernetes.io/docs/reference/generated/kube-apiserver/) for a reference of supported apiserver options.
-
-Below is a list of apiserver options that acs-engine will configure by default:
-
-|apiserver option|default value|
-|---|---|
-|"--admission-control"|"NamespaceLifecycle, LimitRanger, ServiceAccount, DefaultStorageClass, ResourceQuota, DenyEscalatingExec, AlwaysPullImages"|
-|"--authorization-mode"|"Node", "RBAC" (*the latter if enabledRbac is true*)|
-|"--audit-log-maxage"|"30"|
-|"--audit-log-maxbackup"|"10"|
-|"--audit-log-maxsize"|"100"|
-|"--feature-gates"|No default (can be a comma-separated list)|
-|"--oidc-username-claim"|"oid" (*if has AADProfile*)|
-|"--oidc-groups-claim"|"groups" (*if has AADProfile*)|
-|"--oidc-client-id"|*calculated value that represents OID client ID* (*if has AADProfile*)|
-|"--oidc-issuer-url"|*calculated value that represents OID issuer URL* (*if has AADProfile*)|
-
-
-Below is a list of apiserver options that are *not* currently user-configurable, either because a higher order configuration vector is available that enforces apiserver configuration, or because a static configuration is required to build a functional cluster:
-
-|apiserver option|default value|
-|---|---|
-|"--bind-address"|"0.0.0.0"|
-|"--advertise-address"|*calculated value that represents listening URI for API server*|
-|"--allow-privileged"|"true"|
-|"--anonymous-auth"|"false|
-|"--audit-log-path"|"/var/log/apiserver/audit.log"|
-|"--insecure-port"|"8080"|
-|"--secure-port"|"443"|
-|"--service-account-lookup"|"true"|
-|"--etcd-cafile"|"/etc/kubernetes/certs/ca.crt"|
-|"--etcd-certfile"|"/etc/kubernetes/certs/etcdclient.crt"|
-|"--etcd-keyfile"|"/etc/kubernetes/certs/etcdclient.key"|
-|"--etcd-servers"|*calculated value that represents etcd servers*|
-|"--etcd-quorum-read"|"true"|

```

Comment:
My editor removed a lot of erroneous whitespace apparently, but this is the only line that actually changed (was removed.